src/App.js

```diff
@@ -1,41 +1,30 @@
-import logo from './logo.svg';
 import './App.css';
 import PokerTable from "./PokerTable";
+import HandHistory from './HandHistory';
 
 function App() {
-  const history = {
-    cards: {
-      flop: "Ah7d8s",
-      turn: "Ad",
-      river: null
-    },
-    individuals: [
-      {
-        initials: "AC",
-        pot: 600,
-      },
-      {
-        initials: "GH",
-        pot: 1600,
-        hero: true
-      },
-      {
-        initials: "RO",
-        pot: 60,
-      },
-      {
-        initials: "LP",
-        pot: 39292929,
-        current: true
-      }
-    ],
-    actions: [
-      {initials: "RO", action: "fold"}
-    ]
-  }
+  const hand = new HandHistory('KB', 'AdJh');
+  hand.addStack('KB', 100);
```

Comment:
hardly important, but if these returned `this` you could do this as a fluent interface